Kickstarter-iOS/Views/Controllers/SearchViewController.swift

```diff
@@ -16,13 +16,26 @@ internal final class SearchViewController: UITableViewController {
   @IBOutlet fileprivate weak var searchStackView: UIStackView!
   @IBOutlet fileprivate weak var searchTextField: UITextField!
 
+  private let backgroundView = UIView()
+  private let popularLoaderIndicator = UIActivityIndicatorView()
+  private let searchLoaderIndicator = UIActivityIndicatorView()
```

Comment:
it seems that we have two different ways of handling the loading indicator depending on whether it's popular projects loading or search results loading: one uses the background view of the table view and the other uses a table header. i believe it's done that way so that the popular indicator is centered vertically in the screen.

i think we should either handle both of those cases as essentially the same thing, or we should make it clearer how we are handling the two cases.

i prefer the former since i don't think it's necessary to position the indicator differently based on popular vs search loading.